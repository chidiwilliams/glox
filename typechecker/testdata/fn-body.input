var fn = fun (firstName: string, lastName: string): string {
	return firstName - lastName;
};
fn;
