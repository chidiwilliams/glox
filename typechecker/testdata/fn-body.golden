unexpected type: string in {{23 firstName %!s(<nil>)} 8 - %!s(<nil>) {23 lastName %!s(<nil>)}}, allowed: [number]
